__tests__/__unit__/utils.unit.test.ts

```diff
@@ -0,0 +1,79 @@
+/*
+* This program and the accompanying materials are made available under the terms of the *
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+* https://www.eclipse.org/legal/epl-v20.html                                      *
+*                                                                                 *
+* SPDX-License-Identifier: EPL-2.0                                                *
+*                                                                                 *
+* Copyright Contributors to the Zowe Project.                                     *
+*                                                                                 *
+*/
+
+import * as vscode from "vscode";
+import * as utils from "../../src/utils";
+import * as globals from "../../src/globals";
+import { createInstanceOfProfile, createIProfile } from "../../__mocks__/mockCreators/shared";
+import { Profiles } from "../../src/Profiles";
+
+function createGlobalMocks() {
+    const isTheia = jest.fn();
+
+    Object.defineProperty(vscode.window, "showQuickPick", { value: jest.fn(), configurable: true });
+    Object.defineProperty(vscode.window, "createQuickPick", { value: jest.fn(), configurable: true });
+    Object.defineProperty(vscode.window, "showInputBox", { value: jest.fn(), configurable: true });
+    Object.defineProperty(vscode.window, "showErrorMessage", { value: jest.fn(), configurable: true });
+    Object.defineProperty(Profiles, "getInstance", { value: jest.fn(), configurable: true });
+    Object.defineProperty(globals, "ISTHEIA", { get: isTheia, configurable: true });
+
+    return {
+        isTheia
+    };
```

Comment:
I suggest we pass an argument to createGlobalMocks to set up the mocks properly like this:

```suggestion
function createGlobalMocks({isTheia=true} = {}) {
    Object.defineProperty(vscode.window, "showQuickPick", { value: jest.fn(), configurable: true });
    Object.defineProperty(vscode.window, "createQuickPick", { value: jest.fn(), configurable: true });
    Object.defineProperty(vscode.window, "showInputBox", { value: jest.fn(), configurable: true });
    Object.defineProperty(vscode.window, "showErrorMessage", { value: jest.fn(), configurable: true });
    Object.defineProperty(Profiles, "getInstance", { value: jest.fn(), configurable: true });
    Object.defineProperty(globals, "ISTHEIA", { value: isTheia, configurable: true });
    }
```

and that we then use it in tests like this

```js
        const globalMocks = createGlobalMocks({isTheia: true});
```